### PR TITLE
Troi bot daily jams

### PR DESCRIPTION
**Pull request by mayhem**
This PR adds a new function that runs once the recommendations have been generated, so that troi-bot can generate its daily playlists. It includes the following changes:

1. Somewhat unrelated, make collaborator for playlist submission checking more robust.
2. Run troi-bot once recommendations have been generated
3. Generate recommendations daily.
4. Add support for deleting old troi-bot playlists before generating new playlists.

### CODE REVIEW

File: listenbrainz/db/playlist.py [hunk 1]
```diff
@@ -410,21 +417,32 @@ def create(playlist: model_playlist.WritablePlaylist) -> model_playlist.Playlist
...
-        playlist.recordings = insert_recordings(connection, playlist.id, playlist.recordings, 0)
-
-        if playlist.collaborator_ids:
-            add_playlist_collaborators(connection, playlist.id, playlist.collaborator_ids)
-            collaborator_ids = get_collaborators_for_playlists(connection, [playlist.id])
-            collaborator_ids = collaborator_ids.get(playlist.id, [])
-            playlist.collaborators = get_collaborators_names_from_ids(collaborator_ids)
-
-        return model_playlist.Playlist.parse_obj(playlist.dict())
+        with connection.begin():
```
Comment (amCap1712) [inline listenbrainz/db/playlist.py:420]: Adding this delete code to general playlist create method feels wrong. Can we do this elsewhere? Probably just before troi runs in the spark handler or maybe with a --delete-existing flag in troi itself?

**Review by amCap1712 — APPROVED** [review]
Looks good, except that 1 comment.

File: listenbrainz/db/playlist.py (hunk 1 above)
Comment (mayhem) [reply listenbrainz/db/playlist.py:435]: Moved to a separate function as discussed in IRC.

**Comment by mayhem** [comment]
Git tag updated. Once tests pass, its ready to merge.

**Pull request merged**